index.Rmd

```diff
@@ -210,256 +208,115 @@ language_column
+### Using `ggplot` to create a bar plot
 
-### Using `ggplot` to create a scatter plot
+In our data set, we can see that each variable--- higher-level language category (`category`), language (`language`), mother tongue (`mother_tongue`), language spoken most at home (`most_at_home`), etc. ---are all in separate columns. In addition, there is a single row (or observation) for each language. The data are, therefore, in what we call a *tidy data* format. Tidy data is a fundamental concept and will be a significant focus in the remainder of this course: many of the functions from `tidyverse` require tidy data, including the `ggplot` function that we will use shortly for our visualization. We will formally introduce this concept in chapter 3.
 
-Taking another look at our data set below, we can immediately see that the three columns (or variables) we are interested in visualizing---mother tongue, language spoken most at home, and higher level language category---are all in separate columns. In addition, there is a single row (or observation) for each language. 
-The data are therefore in what we call a *tidy data* format. 
-Tidy data is particularly important concept and will be a major focus in the remainder of this course: many of the functions from `tidyverse` require tidy data, 
-including the `ggplot` function that we will use shortly for our visualization. We will formally introduce this concept in chapter 3.
+To answer our data analysis question, we will begin by using the `slice_max` function, which selects rows with the highest values of a variable. Here we want to choose the ten Aboriginal languages most often reported as a mother tongue language. In the function, we need to pass the data frame, the number of rows to select, and the variable to order by. 
 
 ```{r}
-can_lang
+ten_lang <- slice_max(aboriginal_lang, n = 10, order_by = mother_tongue)
+ten_lang
 ```
 
-We will begin with a scatter plot of the `mother_tongue` and `most_at_home` columns from our data frame.
-To create a scatter plot of these two variables using the `ggplot` function, we do the following: 
+We will make a bar plot using the `most_at_home` and `language` columns from our `ten_lang` data frame. To create a bar plot of these two variables using the `ggplot` function, we would the following: 
 
-1. call the `ggplot` function
-2. provide the name of the data frame as the first argument
-3. call the aesthetic function, `aes`, to specify which column will correspond to the x-axis and which will correspond to the y-axis
-4. add a `+` symbol at the end of the `ggplot` call to add a layer to the plot
-5. call the `geom_point` function to tell R that we want to represent the data points as dots/points to create a scatter plot.
+```{r img-ggplot, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Creating a bar plot with the ggplot function", out.width="1100", fig.retina = 2}
```

Comment:
This figure is really awesome and pretty 🤩

Reply:
Thank you! 👍